etc/helm/pachyderm/values.yaml

```diff
@@ -637,3 +638,60 @@ oidc:
   dexCredentialSecretName: ""
   mockIDP: true
   #TODO scopes:
+
+# The proxy is a service to handle all Pachyderm traffic (S3, Console, OIDC, Dex, GRPC) on a single
+# port; good for exposing directly to the Internet.
+proxy:
+  # If enabled, create a proxy deployment (based on the Envoy proxy) and a service to expose it.  If
+  # ingress is also enabled, any Ingress traffic will be routed through the proxy before being sent
+  # to pachd or Console.
+  enabled: false
+  # The number of proxy replicas to run.  1 should be fine, but if you want more for higher
+  # availability, that's perfectly reasonable.  Each replica can handle 50,000 concurrent
+  # connections.  There is an affinity rule to prefer scheduling the proxy pods on the same node as
+  # pachd, so a number here that matches the number of pachd replicas is a fine configuration.
+  # (Note that we don't guarantee to keep the proxy<->pachd traffic on-node or even in-region.)
+  replicas: 1
+  # The envoy image to pull.
+  envoyImage: envoyproxy/envoy:v1.22.0
```

Comment:
I think this should be image.repository, image.tag, image.pullPolicy like we do in other places

Reply:
Will update.

Reply:
also remove the `envoyImage`

Reply:
doh!

Reply:
that is fixed, and now all the tests pass!